saleor/shipping/interface.py

```diff
@@ -39,3 +40,9 @@ def is_external(self) -> bool:
             return str_type == "app"
 
         return False
+
+    @property
+    def graphql_id(self):
+        if self.is_external:
+            return self.id
```

Comment:
So it's actually not `graphql` one?

Reply:
For externals it is:
```
def to_shipping_app_id(app: "App", shipping_method_id: str) -> "str":
    return base64.b64encode(
        str.encode(f"{APP_ID_PREFIX}:{app.pk}:{shipping_method_id}")
    ).decode("utf-8")
```

This piece of code is extracted from the original `resolve_id` method of the `ShippingMethod` object type hence the name